.github/workflows/ci.yml

```diff
@@ -114,10 +114,11 @@ jobs:
         with:
           images: gresearchdev/siembol-${{ matrix.component }}
           flavor: |
-            latest=${{ github.ref == 'refs/heads/master' }}
+            latest=${{ !endsWith(steps.info.outputs.version, 'SNAPSHOT') }}
```

Comment:
I wonder whether the conditions for our special tags should not also restrict them to the `master` branch like before? This is not strictly needed, as we won't publish them anyway, but could be better in case we refactor the pipeline in the future and forget about it.

e.g.
```suggestion
            latest=${{ github.ref == 'refs/heads/master' && !endsWith(steps.info.outputs.version, 'SNAPSHOT') }}
```

same question/remark for every conditions below

Reply:
it makes sense, more readability with this clean condition ⚡

Reply:
Could you please implement this for every condition? 🙏

Reply:
done 🚀  sorry for keeping you wait

Reply:
Could you please implement it for the other 2 `latest=${{ ...` lines?

Reply:
🤦‍♂️my bad, added it now ⚡